web/api/v1/api_test.go

```diff
@@ -314,6 +308,24 @@ func TestEndpoints(t *testing.T) {
 		}
 
 		testEndpoints(t, api, false)
+	}
+
+	// Run all the API tests against a API that is wired to forward queries via
```

Comment:
I find it a bit confusing, but I think it will work.. (: More opinions on those tests would be nice.